project/HmrcBuild.scala

```diff
@@ -59,8 +59,8 @@ object HmrcBuild extends Build {
 private object AppDependencies {
 
   val compile =  Seq(
-    "uk.gov.hmrc" %% "play-json-logger" % "2.1.1",
-    "uk.gov.hmrc" %% "play-authorisation" % "3.1.0"
+    "uk.gov.hmrc" %% "play-json-logger" % "3.1.0",
+    "net.codingwell" %% "scala-guice" % "4.1.1"
```

Comment:
What do we use it for?

Reply:
@tomasz-rosiek This is used for the Guice injection, might not be needed, will check.